packages/theme-ui/README.md

```diff
@@ -216,6 +216,25 @@ To change the underlying component in `Styled`, use the `as` prop.
 <Styled.a as={Link} to='/'>Home</Styled.a>
 ```
 
+## Hooks
```

Comment:
I didn't notice this earlier, but I'm not sure if it makes sense to introduce hooks in the readme since it's meant to be sort of a high-level intro and hooks are a more advanced feature – what do you think @johno?

Reply:
I'd def agree there 👍

Reply:
I've just removed this 👍